src/partner/partner.service.ts

```diff
@@ -90,4 +91,19 @@ export class PartnerService {
 
     return 'Successful';
   }
+
+  async updatePartner(partnerId: string, { active }: UpdatePartnerDto){
```

Comment:
Do we also need to update the associated `partner_access` and `user` table records in this function?

Reply:
Nope just partner! ⭐

Reply:
@eleanorreem Okay! Then I think it's good to go :)